aerogear/mobile-developer-console
# Fix: Create build resets form to initial state, not the empty one

## Motivation
Small fix of create build config functionality. When new client was created, state of create build config form is cleared, but it should be reset to its initial state.

## What
Setting initialState instead of empty state.

## Why
Functionality was unexpected.

## How

## Verification Steps
Add the steps required to check this change. Following an example.
 
1. Go to some mobile app and Build tab
2. Create new build config (Actions menu or button if list of build configs is empty)
3. After build config is created, try to do it again
4. There should be prefilled Jenkins path ./ and branch `master` under advanced source options. It was empty before that.

## Checklist:


## Progress
